pkg/system/reconciler.go

```diff
@@ -67,6 +67,7 @@ type Reconciler struct {
 	NooBaa                    *nbv1.NooBaa
 	ServiceAccount            *corev1.ServiceAccount
 	CoreApp                   *appsv1.StatefulSet
+	CoreAppConfig			  *corev1.ConfigMap
```

Comment:
Indentation

Reply:
Thats weird, in the VSCode it looks indented. 
any idea why here it doesn't?